src/ontoform/transformers/homologue.py

```diff
@@ -1,21 +1,37 @@
+import json
+import subprocess
 from pathlib import Path
+from typing import BinaryIO
+from uuid import uuid4
 
 import polars as pl
 
-def transform(src: Path, dst: Path) -> None:
-    # load the homologues
-    initial = pl.read_json(src)
 
-    # prepare node data
-    inputGenes = pl.DataFrame(
-        initial['genes']
+def transform(src: BinaryIO, dst: BinaryIO) -> None:
+    jq_command = '.genes | {"genes": map({id: .id, name: .name})}'
+
+    # Generate a temporary file path tp store the source file
+    temp_path = Path(f'/tmp/{uuid4()}.json')
+
+    temp_path.write_bytes(src.read())
+
+    # Ejecuta jq con subprocess
+    result = subprocess.run(
+        ['jq', jq_command, temp_path],
+        capture_output=True,
+        text=True
     )
```

Comment:
```suggestion
    # Generate a temporary file path tp store the source file
    with tempfile.NamedTemporaryFile() as temp_file:
        temp_file.write(src.read())
        result = subprocess.run(
            ['jq', jq_command, temp_file.name],
            capture_output=True,
            text=True,
        )
```

It's better to use [`tempfile`](https://docs.python.org/3/library/tempfile.html#tempfile.NamedTemporaryFile) when creating temporary files for security reasons.